lib/liftoff/xcodeproj_helper.rb

```diff
@@ -40,9 +40,19 @@ def set_indentation_level(level, use_tabs)
     def add_script_phases(scripts)
       if scripts
         scripts.each do |script|
-          key, value = script.first
-          puts "Adding shell script build phase '#{value}'"
-          add_shell_script_build_phase(file_manager.template_contents(key), value)
+
+          # -1 means last as index
```

Comment:
Instead of adding this comment, maybe make a constant to hold the value: `LAST_INDEX = -1`